strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/RebuildMavenMetadataCronJobTestIT.java

```diff
@@ -64,38 +61,28 @@ public void init(TestInfo testInfo)
         super.init(testInfo);
     }
 
-    private Set<MutableRepository> getRepositories(TestInfo testInfo)
-    {
-        Set<MutableRepository> repositories = new LinkedHashSet<>();
-        repositories.add(createRepositoryMock(STORAGE0,
-                                              getRepositoryName("rmmcjtit-snapshots", testInfo),
-                                              Maven2LayoutProvider.ALIAS));
-        repositories.add(createRepositoryMock(STORAGE0,
-                                              getRepositoryName("trmir-snapshots", testInfo),
-                                              Maven2LayoutProvider.ALIAS));
-
-        return repositories;
-    }
-
-    @AfterEach
-    public void removeRepositories(TestInfo testInfo)
-            throws Exception
-    {
-        removeRepositories(getRepositories(testInfo));
-    }
-
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class, ArtifactManagementTestExecutionListener.class })
     @Test
-    public void testRebuildArtifactsMetadata(TestInfo testInfo)
+    public void testRebuildArtifactsMetadata(@MavenRepository(storageId = STORAGE0,
+                                                              repositoryId = RMMCJTIT_SNAPSHOTS)
+                                             Repository repository,
+                                             @MavenTestArtifact(repositoryId = RMMCJTIT_SNAPSHOTS,
+                                                                id = "org.carlspring.strongbox:strongbox-metadata-one",
+                                                                versions = { "2.0-20190512.202015-1",
+                                                                             "2.0-20190512.202015-2",
+                                                                             "2.0-20190512.202015-3",
+                                                                             "2.0-20190512.202015-4",
+                                                                             "2.0-20190512.202015-5" },
+                                                                classifiers = { "javadoc",
```

Comment:
use the `CLASSIFIERS` constant here

Reply:
The value for an annotation must be a compile time constant. Declaring the classifiers array as constant and using it inside annotation would fail this test case.

Please let me know if I am wrong.

https://stackoverflow.com/questions/16509065/get-rid-of-the-value-for-annotation-attribute-must-be-a-constant-expression-me

Reply:
yeah, sorry for confusion :)